array-manipulation-2/src/take-right.js

```diff
@@ -0,0 +1,17 @@
+/* exported takeRight */
+
+/*
+loop through array starting at given length - index,
+  check if i is within range of indexes (greater than or equal 0)
+push into output array then return that
+*/
+
+function takeRight(array, count) {
+  var output = [];
+  for (var i = array.length - count; i < array.length; i++) {
```

Comment:
How about `var i = Math.max(0, array.length - count)`?

If you do this, you can remove the need to have a conditional within the loop